CMake/telesculptor-depends.cmake

```diff
@@ -24,11 +22,6 @@ include( telesculptor-utils ) # local utilities
 set(Qt5_MODULES Core Designer UiPlugin Widgets Svg Xml)
```

Comment:
Do I even need to find `Designer`, `UiPlugin`, and `Xml`?  Later I'm only directly linking to `Widgets` and `Svg` and that seems to work.

Reply:
Maybe, maybe not... this might be a work-around for needing to find everything that qtExtensions needs, although since the Designer stuff is strictly plugins and I don't believe exported, it is probably safe to drop at least `Designer` and `UiPlugin`.

I'm not sure about QtXml; I tried removing it from `Qt5_MODULES`, but it's still being linked, possibly via some other dependency.